standalone/ansible/roles/postgres_replication/vars/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+replication_user: replicator
+replication_config_master:
+  - {name: wal_level, value: replica, regex: '#?wal_level = \w+(\s+#.*)'}
```

Comment:
What are these regexes for?

Reply:
They are used to do a find and replace in the config file. Ansible has a [lineinfile](https://docs.ansible.com/ansible/latest/modules/lineinfile_module.html) tool to do that. It's being done here:
https://github.com/simpledotorg/deployment/blob/6cd309452cec50659e78dc23e427684d980331c0/standalone/ansible/roles/postgres_replication/tasks/replica.yml#L25